### ISSUE TITLE
[BUG] Console button doesn't show up

### ISSUE THREAD

**Issue by Havoclivekiller** [TaureHorn/FoundryVTT_ChatConsole#3]
The console button doesn't show up when I add the module to an existing world with a filled up chat. You need to clear chat for the button to show up. Also, clicking the button gives me an error

Uncaught TypeError: Cannot read properties of undefined (reading 'apps')
    at ConsoleManager.render (consoleManager.js:86:24)
    at HTMLButtonElement.<anonymous> (console.js:46:66)
    at HTMLElement.dispatch (jquery.min.js:2:43184)
    at y.handle (jquery.min.js:2:41168)

**Comment by EmoCthulhu** [comment]
Strange... scrolling all the way to the top has it show up for me. I was going to file an issue, that it's really annoying to have to do so, so maybe the console manager could be put on the system settings page instead? I'm about to try it on Vanilla Foundry (I just tested on The Forge first), and see if I get the same console error as you do. Clicking the button for me opened the console manager, so I didn't look at the console. Will update in a minute.

**Comment by EmoCthulhu** [comment]
So here's my update, on both The Forge and Vanilla Foundry clicking the button the first time creates a journal entry properly. After the journal entry is created, you can click the button again and the console manager will come up.

I too, get the console error:

logCompatibilityWarning @ commons.js:2070
foundry.js:68326 Console | Created JournalEntry '_console-data' to store module data
consoleManager.js:86 Uncaught TypeError: undefined. Cannot read properties of undefined (reading 'apps')
[Detected 1 package: console]
    at ConsoleManager.render (consoleManager.js:86:24)
    at HTMLButtonElement.<anonymous> (console.js:46:66)
    at HTMLElement.dispatch (jquery.min.js:2:43184)
    at y.handle (jquery.min.js:2:41168)
commons.js:8807 Foundry VTT | Created JournalEntry with id [mloYUOg3PNvFR5F3]

But it doesn't seem to affect operation. After the journal entry is created and the button now opens the console manager, there is no more error in the Foundry console.

Still think there could be a better place to place the console manager button.

**Comment by TaureHorn** [comment]
I've moved the button to elsewhere in the main chat window. It being inaccessible in a game with any chat history was just a dumb oversight on my part.

I've been unable to recreate the error you've experienced, testing it on three different worlds with three different systems. So I'm unsure what has caused the issue. It's trying to read a document which should have an `apps` key as standard afaik.

Just to be clear, other than there being an error in the console, there's no other issues with functionality? 

If not I can close this issue onces I pushed a new release so you can actually access the button ;)

**Comment by EmoCthulhu** [comment]
Yes, for me at least, the console error doesn't seem to affect
functionality at all.

**Comment by TaureHorn** [comment]
Merge #5, release v1.2.1 fixed the issue with the placement of the button, no scrolling back up needed.

I made other changes that *might* affect the other error, but without knowing why it happened, where it came from or even how to recreate it, that's just a guess. 

If you're on an older version and have problems updating, just uninstall >> reinstall. I ballsed up the `module.json` on the older versions. Should be fine now however
